packages/console/src/widget.ts

```diff
@@ -337,9 +337,14 @@ export class CodeConsole extends Widget {
    *
    * @returns A promise that indicates when the injected cell's execution ends.
    */
-  inject(code: string): Promise<void> {
+  inject(code: string, metadata: JSONObject = {}): Promise<void> {
     let cell = this.createCodeCell();
     cell.model.value.text = code;
+    if (metadata) {
```

Comment:
This is always true if the default is `{}`. How about we just delete the `if` check and let the `for` loop just iterate over an empty object if the default is passed in?